apps/ledger-live-desktop/src/renderer/families/tezos/AccountBodyHeader/index.jsx

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React, { useCallback } from "react";
+import Delegation from "../Delegation";
+import Box from "~/renderer/components/Box";
+import Card from "~/renderer/components/Box/Card";
+import { Trans, useTranslation } from "react-i18next";
+import { useHistory } from "react-router-dom";
+import Text from "~/renderer/components/Text";
+import styled from "styled-components";
+import type { Account } from "@ledgerhq/types-live";
+import Label from "~/renderer/components/Label";
+import Button from "~/renderer/components/ButtonV3";
+import { FeatureToggle } from "@ledgerhq/live-common/featureFlags/index";
+
+type Props = {
+  account: Account,
+  parentAccount: Account | null,
+};
+
+const Wrapper = styled(Box).attrs(() => ({
+  p: 4,
+}))`
+  border-radius: 4px;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const TableContainer = styled(Card)`
+  border-radius: 4px;
+  overflow: hidden;
+`;
+
+const AccountBodyHeader = ({ account, parentAccount }: Props) => {
+  const history = useHistory();
+  const { t } = useTranslation();
+
+  const onNFTClick = useCallback(() => {
+    const value = `/platform/objkt`;
+    history.push({
+      pathname: value,
```

Comment:
```suggestion
    const pathname = `/platform/objkt`;
    history.push({
      pathname,
```

the variable has a more explicit name, makes the code easier to understand